debug/pom.xml

```diff
@@ -24,7 +24,7 @@
   <packaging>pom</packaging>
 
   <properties>
-    <tycho.scmUrl>scm:git:https://github.com/eclipse-platform/eclipse.platform.debug.git</tycho.scmUrl>
+    <tycho.scmUrl>scm:git:https://github.com/eclipse-platform/eclipse.platform.git</tycho.scmUrl>
```

Comment:
It's defined at https://github.com/eclipse-platform/eclipse.platform/blob/master/pom.xml#L27 so I think best is to delete these properties here.

Reply:
Thanks - removed and updated PR title and description and contents to make the same update for other repos that were merged into platform